### ISSUE TITLE
Accessibility: Image Chevrons can't be seen on white images

### ISSUE THREAD

**Issue by edent** [mastodon/mastodon#20474]
### Steps to reproduce the problem

1. When viewing a gallery of images, navigation chevrons appear `<` and `>`
![Screenshot of Mastodon showing pale chevrons](https://user-images.githubusercontent.com/837136/201462389-e3b1a32a-d67c-48e6-914c-1c22d1a63d30.png)
3. On images with a white backgrounds, these are invisible
![Screenshot of Mastodon gallery, but the chevrons can't be seen](https://user-images.githubusercontent.com/837136/201462417-b2cedf18-a8c0-4c70-b8a1-50bff2196d4a.png)

### Expected behaviour

Chevrons should be visible against pale backgrounds

### Actual behaviour

They cannot be seen

### Detailed description

This can be fixed in CSS using [-webkit-text-stroke](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-text-stroke).  Although it is a vendor prefix, it is supported everywhere.

```
.fa-chevron-right::before {
  content: "";
  -webkit-text-stroke: 1px #000b;
}
```

A little transparency is needed so the chevrons don't obscure the images too much:

![Screenshot of Mastodon where the chevrons have a light border image](https://user-images.githubusercontent.com/837136/201462480-a0c32054-80df-4a8e-a4dc-ee6dca50b2e9.png)

I think this only affects https://github.com/mastodon/mastodon/blob/main/app/javascript/styles/mastodon/rtl.scss - so I'll make a PR for that. Let me know if it needs changing elsewhere.

### Specifications

Mastodon: v4.0.0rc3 (mastodon.social)

**Comment by wildeng** [comment]
I’m more of a backend dev, but the PR looks good to me. Just a small question, among the supported browser I didn’t see Firefox for iOS, will it work on that one too?

**Comment by edent** [comment]
Thanks @wildeng. [All browsers on iOS use the Safari rendering engine](https://brucelawson.co.uk/2021/briefing-to-the-uk-competition-and-markets-authority-on-apples-ios-browser-monopoly-and-progressive-web-apps/). So if it works on Safari, it will work on *any* iOS browser.

I've tested it on FF for Android and Linux and it works there.

**Comment by wildeng** [comment]
lgtm

**Comment by trwnh** [comment]
Images seem to not extend to the edges anymore, so the chevrons no longer overlap.
